botfront/imports/ui/components/common/IconButton.jsx

```diff
@@ -9,7 +9,7 @@ const IconButton = React.forwardRef((props, ref) => {
 
     return (
         <Button
-            ref={ref}
+            {...(ref ? { ref } : {})}
```

Comment:
I'm not sure that is needed, if it is undefined it's like not passing the ref right ?

Reply:
you are right, fixed